src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -243,18 +243,23 @@ public synchronized static void startup(NodeId tcmId)
         }
         instance = as;
 
-        replayJournal(as);
+        // If we hit an error during journal replay, we need to mark ourselves unsafe to read, perform full data repair,
+        // and trigger RX before we can continue serving traffic
+        if (replayJournal(as) && ClusterMetadata.current().directory.allJoinedEndpoints().size() > 1)
+            AsyncChains.awaitUninterruptibly(as.node().commandStores().rebootstrap(as.node()));
     }
 
     @VisibleForTesting
-    public static void replayJournal(AccordService as)
+    public static boolean replayJournal(AccordService as)
```

Comment:
I think it is surprising for `true` to mean replay failed. I think we should either use `false`, or introduce an `enum`